challenge_2/style.css

```diff
@@ -0,0 +1,42 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  background: #6592CF;
+  height: 300px;
+  padding: 0;
+  position: absolute;
+  width: 400px;
+}
+
+.boxInside {
+  background: #243D83;
+  height: 150px;
+  margin-left: 50px;
+  margin-top: 75px;
+  position: absolute;
+  width: 300px;
+}
+
+.bigCircle {
+  border: 60px solid #6592CF;
+  border-radius: 100%;
```

Comment:
It's not necessary to put 100%, if you want to accomplish a full circle, with `border-radius: 50%` would be enough 😉